src/org/zaproxy/zap/extension/websocket/client/HandshakeConfig.java

```diff
@@ -35,8 +35,8 @@
 public class HandshakeConfig {
     
     private HttpMessage httpMessage;
-    private boolean followRedirects;
-    private boolean useSessionState;
+    private boolean followRedirects = false;
```

Comment:
The explicit initialisation is not necessary, they are already false by default.